src/diseaseProg.jl

```diff
@@ -561,4 +561,53 @@ function inpFunc_testSpecifications(
     testSpecifications[17:21, "FalsePositiveRate"] .= antibody_FPR_S_to_I4
 
     return testSpecifications
+end
+
+function trFunc_testCapacity(
+    realTime::Date, # time within simulation (day)
+    # PCR capacity - initial
+    testCapacity_pcr_phe_total::Float64 = 1e4,
+    testCapacity_pcr_phe_inflexday::Date = Date("2020-03-25", "yyyy-mm-dd"),
+    testCapacity_pcr_phe_inflexslope::Float64 = 5.0,
+
+    # PCR capacity - increased
+    testCapacity_pcr_country_total::Float64 = 1e5,
+    testCapacity_pcr_country_inflexday::Date = Date("2020-04-25", "yyyy-mm-dd"),
+    testCapacity_pcr_country_inflexslope::Int64 = 10,
+    
+    # Antibody / antigen capacity
+    testCapacity_antibody_country_firstday::Date = Date("2020-04-25", "yyyy-mm-dd"),
+    
+    testCapacity_antibody_country_total::Float64 = 5e6,
+    testCapacity_antibody_country_inflexday::Date = Date("2020-05-20", "yyyy-mm-dd"),
+    testCapacity_antibody_country_inflexslope::Int64 = 20,
+    
+    testCapacity_antigenratio_country::Float64 = 0.7;
+    
+    kwargs...            
+)
+
+    # Returns a dictionary with test names and number available at day "t"
+    outPCR = (
+        #phe phase
+        testCapacity_pcr_phe_total * logistic(Dates.value(Day(realTime-testCapacity_pcr_phe_inflexday))/testCapacity_pcr_phe_inflexslope)
+        +
+        #whole country phase
+        testCapacity_pcr_country_total * logistic(Dates.value(Day(realTime-testCapacity_pcr_country_inflexday))/testCapacity_pcr_country_inflexslope)
+    )
+    
+    
+    if realTime<testCapacity_antibody_country_firstday
+        outAntiTotal = 0.0
+    else
+        outAntiTotal = (
+            testCapacity_antibody_country_total * logistic(Dates.value(Day(realTime-testCapacity_antibody_country_inflexday))/testCapacity_antibody_country_inflexslope)
+		)
+	end
+    
+    return Dict([
```

Comment:
Possibly `nested tuple` will be used instead of `dict`, Please make a comment in `test.jl`  stating the use of `dict`.

Reply:
There is no other use I can see except matching with the python code; so yes, it can be changed to a nested tuple. Would it be better?